client/src/components/OrganisationLink.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const OrganisationLink = ({ organisationName, organisationText, organisationUrl  }) => {
+    return (
+        <div className="organisation-container">
+            <a href={organisationUrl}>
+                <h3 className="bold-font">{organisationName}</h3>
```

Comment:
Looking at the designs vs the component, the font looks too big - the h3 is defaulting to 28px (1.75em) when it should be 20px (1.25em)